src/plexil-adapter/OwInterface.cpp

```diff
@@ -478,96 +478,83 @@ OwInterface::OwInterface ()
 {
```

Comment:
You can remove the explicit initialization of the newly made unique pointers to nullptr, as they are initialized to null by default.